TACC/Core-Portal
# FP-332: adding dashboard link to job history page

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [FP-332](https://jira.tacc.utexas.edu/browse/FP-332)

## Summary of Changes: ##
There is now a link at the top left of the Jobs History table on the Dashboard that links to the Job History page

## Testing Steps: ##
1. Run the code, see that the "View History" link exists
2. Click the "View History" link and make sure it goes to the History/Jobs page.

## UI Photos:

## Notes: ##
